src/containers/FilmFrontpage/Carousel.tsx

```diff
@@ -29,18 +30,12 @@ const StyledSlideContent = styled("div", {
     display: "flex",
     alignItems: "flex-start",
     justifyContent: "space-between",
-    marginInline: "3xlarge",
-    gap: "medium",
-    desktopDown: {
-      gap: "small",
-      marginInline: "medium",
-    },
+    gap: "small",
   },
 });
 
 const CarouselWrapper = styled("div", {
   base: {
-    overflow: "hidden",
```

Comment:
Hvorfor trenger vi ikke å skjule overflow lenger? Vil ikke det fortsatt være et problem dersom vi får flere filmer i karusellen?

Reply:
Jeg kan ikke svare på hvorfor, men når jeg utvider filmene i slideshowet så ser det skjult ut for meg.
Står kanskje igjen noe på ux'en av å ha en karusell på den måten der, men stylingen ser ut til å fungere sånn som jeg forventet i alle fall :shrug: